src/Core/Domain/Product/Exception/CannotUpdateProductException.php

```diff
@@ -31,4 +31,8 @@
  */
 class CannotUpdateProductException extends ProductException
 {
+    /**
+     * When updating product description or description_short fails
+     */
+    const FAILED_UPDATE_DESCRIPTIONS = 1;
```

Comment:
and why do we have "1" when we have multiples of 10 in the exception just above ?

Reply:
copy-paste from chat:
> one more thing I noticed.
 Up until now we didn't use any codes for CannotUpdate{resource}Exception, because we used to update all the obj fields with one command.
The case with product is different, we could add at least one err code for each command. e.g. CannotUpdateProductException::FAILED_UPDATE_DESCRIPTIONS.
Else, we might have problems identifying the err source in controller later. (unless we try some new way of error handling).
Not sure tho, wdyt? (edited) 

this might just allow to identify what operation failed, becaues we will have tons of update commands.

Reply:
> and why do we have "1" when we have multiples of 10 in the exception just above ?

because there is no convention, there are some places to have 10 another have 1, so idk, which is better approach. I agree I should stay with one of them.